Jenkinsfile

```diff
@@ -0,0 +1,259 @@
+
+def config = {
+    owner = 'ksql'
+    slackChannel = '#ksql-alerts'
+    ksql_db_version = "0.6.0-SNAPSHOT"
+    cp_version = "5.5.0-SNAPSHOT" //After done testing this needs to be a beta version.
+    dockerRegistry = '368821881613.dkr.ecr.us-west-2.amazonaws.com/'
+    dockerRepos = ['confluentinc/ksql-cli', 'confluentinc/ksql-rest-app']
```

Comment:
if you look at https://github.com/confluentinc/ksql/pull/3797/files I hack to make sure that we are publishing `ksqldb-cli` and `ksqldb-server`. I can also do this in a follow-up, let me know

Reply:
if you don't mind a follow up PR that would be great.